modules/react/collection/lib/ListBox.tsx

```diff
@@ -64,20 +64,24 @@ export const ListBox = createContainer('ul')({
      */
     Item: ListBoxItem,
   },
-})<ListBoxProps>(({height, maxHeight, ...elemProps}, Element, model) => {
+})<ListBoxProps>(({height, maxHeight, marginY, ...elemProps}, Element, model) => {
+  // We're moving `marginY` to the container to not interfere with the virtualization size. We set
+  // the `marginY` on the Flex to `0` to avoid inaccurate scrollbars
+
   // TODO figure out what style props should go to which `Box`
   return (
     <Box
       ref={model.state.containerRef}
       height={
         height || model.state.isVirtualized ? model.state.UNSTABLE_virtual.totalSize : undefined
       }
+      marginY={marginY}
       maxHeight={maxHeight}
       overflowY={model.state.orientation === 'vertical' ? 'auto' : undefined}
     >
-      <Box as={Element} {...elemProps}>
+      <Flex as={Element} {...elemProps} marginY={0}>
```

Comment:
is there ever a reason consumers MIGHt want to override this?

Reply:
It cannot be overridden. It _will_ mess up the proper rendering of a virtual list

Reply:
You can see that `marginY` is extraced and used above.